sonar-update-center-common/src/test/java/org/sonar/updatecenter/common/UpdateCenterDeserializerTest.java

```diff
@@ -149,6 +150,25 @@ public void should_resolve_ranges() throws IOException {
     }
   }
 
+  // UPC-101
+  @Test(expected = SonarVersionRangeException.class)
```

Comment:
`@Test(expected` is flawed, it's usage is not great

1. if will catch an error raised anywhere in the test method (given the exception is very specific here, I admit impact is limited)
2. no assertion is made on the exception message while it's not just a constant and you could have messed up the `format`

An alternative is usage of JUnit `ExpectedException` Rule (still not perfect, but way better when assertions is right before the method call expected to raise the exception)